src/api/app/models/bs_request_action.rb

```diff
@@ -835,6 +837,16 @@ def commit_details
     package.commit(source_rev) || package.commit
   end
 
+  def toggle_seen_by(user)
+    return unless user
```

Comment:
This code accepts everything which isn't `nil` or `false`, although it could be a bit more robust by checking for the class of `user`. Maybe something like:

```suggestion
    return unless user.is_a?(User)
```